### PR TITLE
Make compatible with latest wagtail version

### PR DESCRIPTION
**Pull request by marcelhekking**
This PR makes the code-base compatible with the latest Wagtail version (2.11.2)

### CODE REVIEW

File: src/wagtail_personalisation/forms.py [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 from importlib import import_module
 
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser
-from django.contrib.staticfiles.templatetags.staticfiles import static
+from django.templatetags.static import static
 from django.test.client import RequestFactory
-from django.utils.lru_cache import lru_cache
+# from django.utils.lru_cache import lru_cache
```
Comment (jberghoef) [inline src/wagtail_personalisation/forms.py:10]: Isn't it better if this line is removed?

**Review by jberghoef — COMMENTED** [review]
One comment on the code. Additionally, could you validate why the Travis CI build is currently failing?

File: src/wagtail_personalisation/forms.py (hunk 1 above)
Comment (marcelhekking) [reply src/wagtail_personalisation/forms.py:10]: Yep. Was the plan. Will remove

**Comment by marcelhekking** [comment]
OK, I will investigate why Travis CI build is failing

File: sandbox/sandbox/settings.py [hunk 2]
```diff
@@ -79,7 +79,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 
-    'wagtail.core.middleware.SiteMiddleware',
+    'tail.contrib.legacy.sitemiddleware.SiteMiddleware',
```
Comment (mikedingjan) [inline sandbox/sandbox/settings.py:82]: I think this is a typo? `wagtail`

Comment (marcelhekking) [reply sandbox/sandbox/settings.py:82]: yep

**Review by blurrah — APPROVED** [review]

**Pull request merged**